protonvpn_cli/connection.py

```diff
@@ -106,21 +100,17 @@ def show_dialog(headline, choices, stop=False):
             get_server_value(servername, "Load", servers)
         ).rjust(3, " ")
 
-        servers_features = []
-        feat = int(get_server_value(servername, 'Features', servers))
-        for bit_flag in features:
-            if (feat & bit_flag) != 0:
-                servers_features.append(features[bit_flag])
+        server_features = get_server_features(servername, servers)
```

Comment:
@marcel-engelke Is this variable rename necessary? It forces 3 other changes that are not code fixes per se. (maybe these little 'extras' are blocking a good review and merge?)